app/javascript/components/organizations-dropdown.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <clickable-dropdown
+    :body-title="dropdownTitle"
+    :no-items-message="noOrganizationsMessage"
+    :items="organizationsWithName"
+    :default-index="defaultOrganizationIndex"
+    @item-clicked="onItemClicked"
+  >
+  </clickable-dropdown>
+</template>
+
+<script>
+import ClickableDropdown from './clickable-dropdown';
+import { PROCESS_NEW_ORGANIZATION } from '../store/action-types';
+import FroggoLocalStorage from '../helpers/local-storage';
+
+export default {
+  props: {
+    organizations: Array,
+    githubLogin: String,
+    teams: Array,
+  },
+  data() {
+    const namedOrganizations = this.organizations;
+    namedOrganizations.forEach((organization) =>
```

Comment:
Arrow function should not return assignment  no-return-assign

Reply:
Acá yo preferiría evitar la mutación:
`const namedOrganizations = this.organizations.map(organization => { ... organization, name: organization.login })`

Cuando haces `const namedOrganizations = this.organizations`, y luego mutas `namedOganizations`, estás mutando `this.organizations` también, la asignación de objetos se hace por referencia en JS, no por valor:
![image](https://user-images.githubusercontent.com/6628178/55833488-a5f7c580-5ae5-11e9-8567-15d83b01dda0.png)